taskcluster/async/asyncclient.py

```diff
@@ -152,6 +147,7 @@ def __del__(self):
             status = response.status
             if status == 204:
                 return None
+            rerr = await response.text()
```

Comment:
Can we rename this variable here and where it's used later in the function?  We're using `rerr` elsewhere in this function for exceptions and here, it's a string.